packages/why3-base/why3-base.0.88.3/opam

```diff
@@ -0,0 +1,66 @@
+opam-version: "1.2"
+maintainer: "guillaume.melquiond@inria.fr"
+authors: [
+  "François Bobot"
+  "Jean-Christophe Filliâtre"
+  "Claude Marché"
+  "Guillaume Melquiond"
+  "Andrei Paskevich"
+]
+
+homepage: "http://why3.lri.fr/"
+license: "GNU Lesser General Public License version 2.1"
+doc: "http://why3.lri.fr/doc/"
+bug-reports: "https://gitlab.inria.fr/why3/why3/issues"
+dev-repo: "https://gitlab.inria.fr/why3/why3.git"
+
+tags: [
+  "deductive"
+  "program verification"
+  "formal specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+]
+available: [ ocaml-version >= "4.02.3" ]
+
+# Jessie3 (frama-c plugin) is *disabled* because it is not ready
+build: [
+  ["./configure" "--prefix" prefix "--disable-frama-c"
+                 "--disable-ide" { !conf-gtksourceview:installed }]
+  [make "-j%{jobs}%" "all" "opt" "byte"]
+]
+
+install: [make "install" "install-lib"]
+
+remove: [
+  ["rm" "%{bin}%/why3"]
+  ["rm" "-r" "%{lib}%/why3"]
+  ["rm" "-r" "%{share}%/why3"]
+]
+
+build-doc: [
+  [make "doc" "stdlibdoc" "apidoc"]
+  [make "DOCDIR=%{doc}%/why3" "install-doc"]
+]
+
+depends: [
+  "ocamlfind"
```

Comment:
Does it depends on the ```Findlib``` library ? If not, this should probably have the ```{build}``` attribute.

Reply:
Good point, I will modify it.

I need to rework the `opam` file anyway, since some unfortunate conflict with Opam's directory hierarchy causes some Why3 files to not be properly installed. So closing for now.

Reply:
I'm curious, what was the problem ?

Reply:
Why3's `Makefile` (incorrectly) expects `$LIBDIR` (as computed by `autoconf`) and `ocamlfind printconf destdir` to be two separate directories (which is usually the case for a user compiling from source). Some recent minor changes to Why3's `Makefile` have made apparent a data race that causes some files to not be installed when both directories happen to be the same, as it is the case with Opam.

Reply:
I see. Thanks.